### ISSUE TITLE
Datetime axis ticks get labeled as numeric (regression)

### ISSUE THREAD

**Issue by frtennis1** [uwdata/mosaic#499]
This snippet used to format the x-axis ticks as datetimes (as desired), but it seems to have recently changed to format them as numeric (not desired),

```
df = pd.DataFrame({
    'ts': pd.date_range('2023-05-25 09:30', freq='1min', periods=100), 
    'y_val': np.cumsum(np.random.normal(size=100))
})

spec = {
    "plot": [
        {
            "mark": "lineY",
            "data": {"from": "df"},
            "x": "ts",
            "y": "y_val",
        }
    ],
}
MosaicWidget(spec, data={"df": df})
```

<img width="674" alt="image" src="https://github.com/user-attachments/assets/f02afff2-b19d-47a7-bfc2-60611b84e57b">

In case it helps: I ran a git bisect and found that the first commit with the undesired behavior is 8b5fd700b1a460f498264cb859eb939d82341bc3 (the arrow js to flechette migration). Unfortunately, it wasn't obvious to me what about that commit caused the change in behavior.

**Comment by jheer** [comment]
Thanks for this! The widget wasn't using the same Arrow decoding settings as the rest of Mosaic, this is now fixed in #499. The default for Mosaic is to map date-valued Arrow data to JS `Date` objects, which results in Observable Plot adjusting the axis ticks accordingly. Please let me know if this doesn't resolve the issue for you.

p.s. in case you are ever intentionally working with numeric timestamp data (not `Date` objects), you can also set the plot scale type to `utc` to force interpretation as dates and get corresponding tick labels.

**Comment by frtennis1** [comment]
Can confirm that did the trick for me, thank you very much!
